bool enable
bool disable
bool increment
bool decrement
bool up
bool down
bool left
bool right
bool home

